CMakeLists.txt

```diff
@@ -202,66 +212,3 @@ install(FILES
     "${CMAKE_CURRENT_BINARY_DIR}/VexConfigVersion.cmake"
     DESTINATION lib/cmake/Vex
 )
-
-# Clang Format and Clang Tidy Setup
```

Comment:
Running clang format and clang tidy upon cmake build doesn't make much sense in hindsight...
I removed this.